src/test/java/edu/cornell/kfs/module/cam/fixture/EntryFixture.java

```diff
@@ -2,10 +2,13 @@
 
 import org.kuali.kfs.gl.businessobject.Entry;
 
+import edu.cornell.kfs.module.cam.CuCamsTestConstants;
+
 public enum EntryFixture {
 	
-	VCM_ONE("5319793", "CM"), VCM_TWO("5686500", "CM"), VCM_THREE("0","CM"),
-	PREQ_ONE("5773686","PREQ"), PREQ_TWO("5773687", "PREQ"), PREQ_THREE("0", "PREQ");
+	VCM_ONE(CuCamsTestConstants.DOC_5319793, "CM"), VCM_TWO(CuCamsTestConstants.DOC_5686500, "CM"), VCM_THREE("0", "CM"),
```

Comment:
Ok, I'll come up with better names for these enum constants, though I may have to alter other unit tests as well if they rely on these.